images/nginx-1.25/README.md

```diff
@@ -1,3 +1,47 @@
 NGINX 1.25 base image
 
-**Don't use in production!!!**
\ No newline at end of file
+**Don't use in production!!!**
```

Comment:
```suggestion
```

I guess normal users do not have a choice here. Also v1.25.x itself should be fine.